project-management/src/main/java/life/qbic/projectmanagement/application/measurement/MeasurementService.java

```diff
@@ -414,10 +312,55 @@ private Map<ProteomicsMeasurement, Collection<SampleIdCodeEntry>> mergeByPool(
     return metadataMap;
   }
 
+  private Map<NGSMeasurement, Collection<SampleIdCodeEntry>> mergeByPoolNGS(
+      Map<String, List<NGSMeasurementMetadata>> groupedMetadata, ProjectId projectId) {
+    Map<NGSMeasurement, Collection<SampleIdCodeEntry>> metadataMap = new HashMap<>();
+    for (var metadataGroup : groupedMetadata.entrySet()) {
+      metadataMap.putAll(buildNGS(metadataGroup.getValue(), projectId));
+    }
+    return metadataMap;
+  }
+
+  private Map<NGSMeasurement, Collection<SampleIdCodeEntry>> buildNGS(
+      List<NGSMeasurementMetadata> metadataList, ProjectId projectId) {
+    Map<SampleCode, SampleIdCodeEntry> sampleIdLookupTable = buildSampleIdLookupTable(metadataList);
+    var sampleCodes = sampleIdLookupTable.keySet();
+    var specificMetadata = createSpecificMetadataNGS(metadataList, sampleIdLookupTable);
+    var assignedMeasurementCode = MeasurementCode.createNGS(sampleCodes.iterator().next().code());
+    var firstMetadataEntry = metadataList.get(0);
+
+    var organisationQuery = organisationLookupService.organisation(
+        firstMetadataEntry.organisationId());
+    if (organisationQuery.isEmpty()) {
+      throw new MeasurementRegistrationException(ErrorCode.UNKNOWN_ORGANISATION_ROR_ID);
+    }
+
+    var instrumentQuery = resolveOntologyCURI(firstMetadataEntry.instrumentCURI());
+    if (instrumentQuery.isEmpty()) {
+      throw new MeasurementRegistrationException(ErrorCode.UNKNOWN_ONTOLOGY_TERM);
+    }
+
+    var method = new NGSMethodMetadata(instrumentQuery.get(), firstMetadataEntry.facility(),
+        firstMetadataEntry.sequencingReadType(), firstMetadataEntry.libraryKit(),
+        firstMetadataEntry.flowCell(), firstMetadataEntry.sequencingRunProtocol());
+
+    NGSMeasurement measurement;
+    if (firstMetadataEntry.assignedSamplePoolGroup().isPresent()) {
+      measurement = NGSMeasurement.createWithPool(projectId,
+          firstMetadataEntry.assignedSamplePoolGroup().get(), assignedMeasurementCode,
+          organisationQuery.get(), method, specificMetadata);
+    } else {
+      measurement = NGSMeasurement.createSingleMeasurement(projectId, assignedMeasurementCode,
+          organisationQuery.get(), method, specificMetadata.get(0));
+    }
```

Comment:
For proteomics there is no difference here. In both cases the `create` method is called. What is the difference to the NGS case?

Reply:
That's a very good question! I asked SF about this, the difference is that during the NGS case we realised that a more stringent differentiation between pooling and singular measurement makes sense due to a difference in mandatory properties. 
He suggested to avoid bloating this PR with changes to Proteomics however so i won't change this in this PR.